server.go

```diff
@@ -1177,6 +1182,58 @@ func startWebgo() {
 	router.Start()
 }
 
+// Goyave
+func goyaveHandler(r *goyave.Response, req *goyave.Request) {
+	if cpuBound {
+		pow(target)
+	} else {
+
+		if sleepTime > 0 {
+			time.Sleep(sleepTimeDuration)
+		} else {
+			runtime.Gosched()
+		}
+	}
+	r.String(http.StatusOK, "hello")
+}
+
+func getGoyaveRoutes(router *goyave.Router) {
+	router.Get("/hello", goyaveHandler)
+}
+
+func startGoyave() {
+	cfg := []byte(`
+	{
+		"server": {
+			"host": "127.0.0.1",
```

Comment:
use 0.0.0.0 as host
use port variable as this port
Do not use hardcode because we may test in different server and port